Airports/EGGD/Fixes.txt

```diff
@@ -6,3 +6,4 @@ GD09F N051.22.33.790 W002.53.09.690
 GD27F N051.23.20.550 W002.33.08.380
 ROTLU N051.18.30.710 W002.47.02.500
 SOMOT N051.17.33.000 W002.59.53.000
+EMPAS N051.22.16.220 W003.00.19.640
```

Comment:
```suggestion
```

EMPAS is already defined in line 4

Reply:
D'oh! Good spot, I've removed the duplicate fix